### PR TITLE
Issue6204: Define fields for archiving of meetings

### PR DESCRIPTION
**Pull request by r-peschke**
@FinnStutzenstein : Please make this job required for merging pull requests

### CODE REVIEW

File: docs/models.yml
Comment (FinnStutzenstein) [inline docs/models.yml:99]: One question here: Is there a way to disable the limiting (Would be good to have!)? Whats about the values 0, -1, ... here. I think you should set a minimum.

File: docs/models.yml
Comment (FinnStutzenstein) [inline docs/models.yml:464]: invalid attribute

File: docker/initial-data.json
Comment (FinnStutzenstein) [inline docker/initial-data.json:18]: Please set "no limit" for the initial data.

**Review by FinnStutzenstein — CHANGES_REQUESTED** [review]

File: docs/models.yml
Comment (r-peschke) [reply docs/models.yml:99]: > One question here: Is there a way to disable the limiting (Would be good to have!)? Whats about the values 0, -1, ... here. I think you should set a minimum.

Values below 1 disable the limit, see last point in [impact on backend service](https://github.com/OpenSlides/OpenSlides/wiki/MeetingArchived#impact-on-backend-service)

File: docs/models.yml
Comment (r-peschke) [reply docs/models.yml:464]: removed the default value, must be set in software to organization id

File: docker/initial-data.json
Comment (r-peschke) [reply docker/initial-data.json:18]: Done, set to 0 (=no_limit

**Comment by FinnStutzenstein** [comment]
One thing:

we have the convention that all id fields must end with `_id` or `_ids`. This is expected, e.g. in the client. The client *can* adopt other naming schemes, but having this naming scheme allows at a quick glance to identify relations vs. non-relations.

I think about adding it to the modelsvalidator now...

File: docs/models.yml
Comment (FinnStutzenstein) [reply docs/models.yml:99]: Please set a minimum of -1

**Review by FinnStutzenstein — CHANGES_REQUESTED** [review]
Please rebase, so #6206 is included

I also wrote a comment about the naming of the relations.

File: docker/initial-data.json
Comment (FinnStutzenstein) [inline docker/initial-data.json:1]: Please undo all formatting changes.

Comment (FinnStutzenstein) [reply docker/initial-data.json:1]: Also in the example-data.json

File: docs/models.yml
Comment (FinnStutzenstein) [inline docs/models.yml:103]: please add `minimum: 0`

**Review by FinnStutzenstein — APPROVED** [review]
One small remark. Before we merge this, we have to decide if we want an initial migration or not (https://github.com/OpenSlides/openslides-backend/pull/901)

File: docs/models.yml
Comment (r-peschke) [reply docs/models.yml:103]: Done

**Comment by FinnStutzenstein** [comment]
Fith fixint the initial data to MI 1, you should remove the check from the CI and add another one:
- get the md5sum of the current initial data and assert in the CI, that the checksum does not change -> detect (accidental) modifications to this file

File: .github/workflows/md5_check_initial-data.yml
Comment (FinnStutzenstein) [inline .github/workflows/md5_check_initial-data.yml:1]: Please add a comment that this is only for protecting against unwanted/unintended edits. But if one is sure to change this file it is allowed!

**Review by FinnStutzenstein — APPROVED** [review]
CI looks good, one comment

File: .github/workflows/md5_check_initial-data.yml
Comment (jsangmeister) [inline .github/workflows/md5_check_initial-data.yml:11]: Can be removed

**Review by jsangmeister — COMMENTED** [review]
Rest looks good. Is the update of the backend intended?

File: .github/workflows/md5_check_initial-data.yml
Comment (r-peschke) [reply .github/workflows/md5_check_initial-data.yml:1]: done

File: .github/workflows/md5_check_initial-data.yml
Comment (r-peschke) [reply .github/workflows/md5_check_initial-data.yml:19]: done

**Review by jsangmeister — APPROVED** [review]

**Comment by FinnStutzenstein** [comment]
The linked backend version has a broken `cleanup-standalone.sh`. Please use the current commit (6218992fe4512b8d00242131de4ce952669ab11a)

**Comment by r-peschke** [comment]
> Rest looks good. Is the update of the backend intended?

yes:)

**Review by FinnStutzenstein — APPROVED** [review]

**Pull request merged**
